pyt/interprocedural_cfg.py

```diff
@@ -404,47 +552,24 @@ def visit_Call(self, node):
         last_attribute = _id.rpartition('.')[-1]
         if definition:
             if isinstance(definition.node, ast.ClassDef):
-                self.add_builtin(node)
+                self.add_blackbox_or_builtin_call(node)
             elif isinstance(definition.node, ast.FunctionDef):
                 self.undecided = False
                 self.function_return_stack.append(_id)
-                return self.add_function(node, definition)
+                return self.process_function(node, definition)
             else:
                 raise Exception('Definition was neither FunctionDef or ' +
                                 'ClassDef, cannot add the function ')
         elif last_attribute not in NOT_A_BLACKBOX:
-            return self.add_blackbox_call(node)
-
-        return self.add_builtin(node)
-
-    def add_class(self, call_node, def_node):
```

Comment:
This was dead-code, but we should figure out where we're at with class support and write more tests.